CredentialProvider.Microsoft/Util/EnvUtil.cs

```diff
@@ -21,6 +22,7 @@ public static class EnvUtil
 
         public const string SupportedHostsEnvVar = "NUGET_CREDENTIALPROVIDER_VSTS_HOSTS";
         public const string SessionTimeEnvVar = "NUGET_CREDENTIALPROVIDER_VSTS_SESSIONTIMEMINUTES";
+        public const string TokenTypeEnvVar = "NUGET_CREDENTIALPROVIDER_VSTS_TOKENTYPE";
```

Comment:
Please add this env var to the EnvironmentVariableHelp text in Resources file. Also makes sense to update it in the readme